lib/plugins/aws/deploy/lib/check-for-changes.js

```diff
@@ -255,6 +255,8 @@ module.exports = {
     const partition = account.partition;
 
     const functionNames = await this.serverless.service.getAllFunctions();
+
+    // #7747, no tests for this file, so did not remove _.flatten()
```

Comment:
How did you determine that there are no tests for this file? There are pretty new tests that cover this logic: https://github.com/serverless/serverless/blob/e7ad5aa268b7b0e10df2c6199e4fd93415de3c30/test/unit/lib/plugins/aws/deploy/lib/check-for-changes.test.js#L1146

Reply:
> pretty new tests

That's probably it. I see the tests on the latest sync. Let me see about changes to this file.

Reply:
I added the change in the latest, and testing looks good.